### ISSUE TITLE
[Feature Request] Include list of all apps in backups

### ISSUE THREAD

**Issue by JeremyStepp** [machiav3lli/oandbackupx#314]
**Is your feature request related to a problem? Please describe.**
Many apps don't need any data backed up, and it would be impractical to store all of their APKs.

**Describe the primer solution**
It would be extremely useful if a list of all installed apps could be included with backups.

**Comment by machiav3lli** [comment]
Please elaborate more. Both points are blurry.
* What types of apps don't need any data backed up?
* What do APKs have to do with that?
* Why to create such a list - for what use case?

**Comment by JeremyStepp** [comment]
- Ones without any important data or configuration such as a gallery or VPN application.
- It is also often not important to store the APK for these apps, because they can just be redownloaded from an app store.
- The use case is when someone wants to conserve backup space, but have a lot of apps they might not remember, but would want again on a new device. There are also apps like map applications that might have tons of data downloaded that isn't important to back up since it can be redownloaded.

Basically for me some of the most important data to back up is what useful apps I found rather than the data stored in the apps. Since OAndBackupX already generates a list of device apps it seems like it would be fairly trivial to optionally export that to a text file with backups. It could also help when people forget to add newer apps to their backup lists that might have been completely forgotten about otherwise.

**Comment by machiav3lli** [comment]
For a lot of users that don't have flat internet or don't use closed source stores it just makes sense to do such backups. So your logical conclusion is faulty, nevertheless the use case is legitim and would be really useful as far as such a list has an automated use (which isn't the situation for now).

That said, I added this feature to my TODO list.
